repository/BaselineOfSeaside3.package/BaselineOfSeaside3.class/instance/baselinepharo..st

```diff
@@ -49,9 +49,16 @@ baselinepharo: spec
 				package: 'Seaside-Pharo-Flow' with: [ spec requires: #('Seaside-Flow' 'Seaside-Pharo-Continuation') ];
 				package: 'Seaside-Tests-Pharo-Continuation' with: [ spec requires: #('Seaside-Pharo-Continuation') ];
 				package: 'Seaside-Tests-Core' with: [ spec includes: #('Seaside-Tests-Pharo-Core') ];
-				package: 'Seaside-Tests-Pharo-Core' with: [ spec requires: #('Seaside-Pharo-Core' 'Seaside-Tests-Core') ].
+				package: 'Seaside-Tests-Pharo-Core' with: [ spec requires: #('Seaside-Pharo-Core' 'Seaside-Tests-Core') ] ].
 
-			"Spec based tools"
-			spec
+	spec
+		for: #(#'pharo4.x' #'pharo5.x' #'pharo6.x' #'pharo7.x')
+		do: [ spec
 				package: 'Seaside-Tools-Core' with: [ spec includes: #('Seaside-Pharo-Tools-Spec') ];
-				package: 'Seaside-Pharo-Tools-Spec' with: [ spec requires: #('Seaside-Tools-Core') ] ]
\ No newline at end of file
+				package: 'Seaside-Pharo-Tools-Spec' with: [ spec requires: #('Seaside-Tools-Core') ] ].
+
+	spec
+		for: (GRPlatform current pharoVersionsFrom: 8)
```

Comment:
shouldn't this be `self pharoVersionsFrom: 8` ? 
(not sure if I like this way, but for sure it's where the travis builds fail on...)

Reply:
@jecisc if possible, could you just change it to `pharo8`? I prefer to manually update for new versions rather than trying to figure out later on if some package should have a `self pharoVersionsBetween: xx and: yy` method :)